config/examples/web_hooks.yml

```diff
@@ -2,16 +2,15 @@ base: &default
   perform_deliveries: <%= ENV.fetch('WEBHOOK_PERFORM_DELIVERIES', '0') == '1' %>
   sanitized_url: <%= ENV.fetch('WEBHOOK_SANITIZED_URL', 'http://127.0.0.1/') %>
 
-production:
-  <<: *default
-
-preview:
-  <<: *default
+development:
+  perform_deliveries: true
+  sanitized_url: 'http://localhost/ping'
```

Comment:
`localhost` and `127.0.0.1` can be different. On modern linux, it usually prefers the ipv6 one `::1`. I don't know if this matters for us.

Reply:
why not `<<: *default` and then only override `perform_deliveries`?